src/identity/types.ts

```diff
@@ -5,10 +5,14 @@ import type { Address, Chain } from 'viem';
  * Note: exported as public Type
  */
 export type AddressReact = {
-  address?: Address | null; // The Ethereum address to render.
-  className?: string; // Optional className override for top span element.
-  isSliced?: boolean; // Determines if the displayed address should be sliced.
-  hasCopyAddressOnClick?: boolean; // Defaults to true. Optional boolean to disable copy address on click functionality.
+  /** The Ethereum address to render. */
+  address?: Address | null;
+  /** Optional className override for top span element. */
+  className?: string;
+  /** Determines if the displayed address should be sliced. */
+  isSliced?: boolean;
```

Comment:
we should say that we default to true, esp because as a dev i wouldn't assume this would default to true